subcommands/targets/tag.go

```diff
@@ -32,33 +35,55 @@ func init() {
 	}
 	cmd.AddCommand(tagCmd)
 	tagCmd.Flags().StringVarP(&tagTags, "tags", "T", "", "comma,separate,list")
+	tagCmd.Flags().BoolVarP(&tagAppend, "append", "", false, "Append the given tags rather than set them")
```

Comment:
should this be called appendtag or append_tag to make it clearer?

Reply:
I didn't have a strong opinion, but it felt redundant. Its the *tag* command. Feels a bit like saying "ATM machine", but some people prefer that.

Reply:
we can let Mike chime in on that

Reply:
@Tim-Anderson the full command would look like this:
```
fioctl targets tag --append <tag> --by-version 42
```

When I read that back to myself it makes perfect sense.